tests/depth_average_underres.prm

```diff
@@ -0,0 +1,112 @@
+# Test the 'depth average' postprocessor on a coarse mesh version of
+# convection_box.prm. This test makes no sense since we average over
+# 100 depth intervals but we only compute on a 4x4 mesh -- several of
+# the depth intervals will receive no quadrature points
+
+set Dimension                              = 2
+
+set Use years in output instead of seconds = false
+set End time                               = 0.5
```

Comment:
from what test is this copied from? (can you add this to the beginning of this file as a comment please)?

Can you set a smaller end time and update the results?